tests/v2/validation/scale/README.md

```diff
@@ -0,0 +1,36 @@
+# Scale
```

Comment:
I am unsure of this design for `scale`. In a separate PR, we are adding support for deleting/scaling K3S/RKE2 clusters. In that PR, I am refactoring test cases that are specific to each use case.

For the scaling part, I think these tests you're adding make sense to fit into that refactoring as opposed to what's going on here.

Reply:
So this is something we had a meeting about while you were out. Basically speaking we want to keep only provisioning tests in the provisioning testing package and everything else out. So only tests that actually test the provisioning actions/calls. One reason being avoiding creating an absurd amount of clusters test cases that are actually testing something else. Ie) scaling, node deletion. What are your concerns? Is it the actual test cases themselves?

Reply:
Got it, that makes sense! I was more so stating that we should limit the redundancy and overhead in areas. Which it sounds like is the point here.

So once this is merged, we would essentially take the scaling PR and add the scale tests into here, am I understanding correctly?

Reply:
correct